<launch>   
    <arg name="filename_load" default="first_pose.yaml"/>
    <arg name="filename_save" default="base_learn_poses.yaml"/>
    
    <node name="pose_calculator" type="calculate_learnposes_node.py" pkg="learn_object" output="screen">
        <param name="filename_load_pose"    value="$(find learn_object)/poses/$(arg filename_load)"/>
        <param name="filename_save_poses"   value="$(find learn_object)/poses/$(arg filename_save)"/>
        <param name="source_frame"          value="base_link"/>
        <param name="target_frame"          value="map"/>
        <param name="number_of_images"      value="5"/>
        <param name="angle_range"           value="180"  />
        <param name="distance"              value="1.5"/>
    </node> 
</launch>

